vibrateonstart/obj/VibrateOnStart.bundle/VibrateOnStart-iPad.plist

```diff
@@ -221,22 +221,6 @@
 		<dict>
 			<key>cell</key>
 			<string>PSGroupCell</string>
-			<key>footerText</key>
```

Comment:
Hmmm, that's actually very strange. I never even touched the preferences, so I did not intend to remove the paypal things. I don't know why that was removed, I have no problem with the dev having a donate button.